io.edgehog.devicemanager.LedBehavior.json

```diff
@@ -0,0 +1,15 @@
+{
+  "interface_name": "io.edgehog.devicemanager.LedBehavior",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "properties",
+  "ownership": "server",
+  "mappings": [
+    {
+      "endpoint": "/%{led_id}/behaviour",
+      "type": "string",
+      "description": "Enum describing the behavior of the given led. Possible values: [ OFF | ON | BLINK | DBLINK | SBLINK]",
```

Comment:
We are using PascalCase for this kind of strings, such as Blink and DoubleBlink. Also I would write "DoubleBlink" instead of "DBLINK".

Reply:
I'm not sure we should support Off (hence On). Is disabling the always-on "connected to the cloud" LED a feature we would like to support?

Reply:
description here still requires to be updated.